app/views/support_interface/application_forms/_application_navigation.html.erb

```diff
@@ -1,18 +1,9 @@
 <% content_for :before_content do %>
-  <%= render BreadcrumbComponent.new(items: [
-    {
-      text: 'Candidates',
-      path: support_interface_applications_path,
-    },
-    {
-      text: @application_form.candidate.email_address,
-      path: support_interface_candidate_path(@application_form.candidate),
-    },
-    {
-      text: @application_form.support_reference,
-      path: support_interface_application_form_path(@application_form),
-    },
-  ].concat(title != 'Details' ? [{ text: title }] : [])) %>
+  <%= govuk_breadcrumbs({
+    'Candidates': support_interface_applications_path,
+    "#{@application_form.candidate.email_address}": support_interface_candidate_path(@application_form.candidate),
```

Comment:
I think `@application_form.candidate.email_address => support_interface_candidate_path(@application_form.candidate)` is preferred style here (though Rubocop doesn't seem to be bothered either way).

Reply:
Updated to use preferred style across all breadcrumbs.